Documentation/Blazorise.Docs/Pages/Docs/Components/Typographies/TypographyPage.razor

```diff
@@ -132,6 +132,16 @@
     <DocsPageSectionSource Code="TypographyTextSizeExample" />
 </DocsPageSection>
 
+<DocsPageSection>
+    <DocsPageSectionHeader Title="Text Decoration">
+        Blazorise allows you to style text with various decorations. You can use underline to place a line beneath the text, line-through to strike through it, or overline to add a line above. To remove any decorations, use the none option. These styles help enhance and highlight text as needed.
```

Comment:
I actually like the bullet point approach with the description for each one.
Thoughts?

Reply:
The example already shows it good IMO.

Reply:
I meant the text not the code example